internal/process/process.go

```diff
@@ -19,6 +19,7 @@ const (
 	entityTypeResourcePool = "ResourcePool"
 	entityTypeVm           = "Vm"
 	entityTypeDatastore    = "Datastore"
+	entityTypeSnapshotVm   = "SnapshotVm"
```

Comment:
this confuses me a bit since there is no entity for snapshot

Reply:
I will change the name!